src/views/pages/reports/reports-page.js

```diff
@@ -37,14 +37,14 @@ export class ReportsPage extends Component {
     this.props.loadTasks(projectUrl);
     if (!this.props.selectedProject) {
       // Load the project
-      this.props.selectProjectFromUrl();
+      this.props.initProject()
```

Comment:
Should redirect to the main screen in case the project still not exists (review my comment in initProject)

Reply:
In a second thought, I think you are correct (see my comment to your initProject review)